src/composables/hover-card.js

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2024 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+*/
+
+// useHoverCard() listens for mouse events on an element. If the user hovers
+// over the element for sufficiently long, a hover card will be shown next to
+// the element.
+
+import { inject, onBeforeUnmount } from 'vue';
+
+import useEventListener from './event-listener';
+import { watchSync } from '../util/reactivity';
+
+// Set this to `true` to prevent the hover card from being hidden on mouseleave.
+// That can be useful during local development, e.g., when iterating on styles.
+const preventHide = false;
+
+export default (anchorRef, type, data) => {
+  const hoverCard = inject('hoverCard');
+  let timeoutId;
+  useEventListener(anchorRef, 'mouseenter', () => {
+    timeoutId = setTimeout(
+      () => {
+        hoverCard.show(anchorRef.value, type, data());
+        timeoutId = null;
+      },
+      350
+    );
+  });
+  const hide = (anchor = anchorRef.value) => {
+    if (hoverCard.anchor === anchor) hoverCard.hide();
+
+    if (timeoutId != null) {
+      clearTimeout(timeoutId);
+      timeoutId = null;
+    }
+  };
+  useEventListener(anchorRef, 'mouseleave', () => {
```

Comment:
Nice to have: I wish if hover cards don't hide on mouse leaving the anchor link and entering into the hover card. 

One way to achieve that is to append `popover` inside the anchor instead of `'body'` (`src/components/popover.vue:87`).

Reply:
I like this idea. If the user could enter the hover card, then we could show tooltips over truncated text. That said, the release criteria says the opposite:

> I should not be able to hover on or interact with the infocard.

How about we discuss more in the meeting tomorrow?

Reply:
We decided that we wouldn't make this change for v2024.3, but I'm going to file a follow-up issue about this.